### ISSUE TITLE
Windows wsl mysql error

### ISSUE THREAD

**Issue by khanalpride** [genesisweb/valet-linux-plus#17]
when doing `valet install` it goes well till mysql-server and fails
any solution?

I am on Windows 10, wsl, tried installing on ubuntu 20.04

```
[mysql-server] is not installed, installing it now via Apt... 🍻
Reading package lists...
Building dependency tree...
Reading state information...
mysql-server is already the newest version (8.0.19-0ubuntu5).
0 upgraded, 0 newly installed, 0 to remove and 13 not upgraded.
2 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up mysql-server-8.0 (8.0.19-0ubuntu5) ...
invoke-rc.d: could not determine current runlevel
 * Stopping MySQL database server mysqld                                 [ OK ]
Renaming removed key_buffer and myisam-recover options (if present)
Cannot open /proc/net/unix: No such file or directory
Cannot stat file /proc/1/fd/5: Operation not permitted
Cannot stat file /proc/1/fd/10: Operation not permitted
Cannot stat file /proc/1/fd/6: Operation not permitted
Cannot stat file /proc/19236/fd/7: Operation not permitted
Cannot stat file /proc/19236/fd/10: Operation not permitted
Cannot stat file /proc/19236/fd/5: Operation not permitted
Cannot stat file /proc/25883/fd/11: Operation not permitted
Cannot stat file /proc/25884/fd/13: Operation not permitted
Cannot stat file /proc/25885/fd/15: Operation not permitted
Cannot stat file /proc/25886/fd/17: Operation not permitted
mysqld will log errors to /var/log/mysql/error.log
mysqld is running as pid 26031
sleep: cannot read realtime clock: Invalid argument
dpkg: error processing package mysql-server-8.0 (--configure):
 installed mysql-server-8.0 package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of mysql-server:
 mysql-server depends on mysql-server-8.0; however:
  Package mysql-server-8.0 is not configured yet.

dpkg: error processing package mysql-server (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
Errors were encountered while processing:
 mysql-server-8.0
 mysql-server
E: Sub-process /usr/bin/dpkg returned an error code (1)


In Apt.php line 74:

  Apt was unable to install [mysql-server].


install [--ignore-selinux]
```

**Comment by uttamrabadiya** [comment]
Hey @khanalpride 

Looks like you have MySQL in your windows 10, I haven't got chance to try it on my Windows system but I think this may can work for you.

1 stop mysql server on the windows10 ( or change the default port? i have not tried)
2 run `valet install` command again
3 re-open wsl, `sudo service mysql stop` and `sudo dpkg --configure -a`

Source of answer:
https://github.com/Microsoft/WSL/issues/3896

Thank you.

**Comment by khanalpride** [comment]
basically there is no mysql installation on my windows. I will try few more things and see if that works. and write here when I have some solution.

**Comment by dvmunjapara** [comment]
@khanalpride 
Seems like you have figure out the solution for this, feel free to share it here. 
If you haven't get what you have been looking for at Valet for windows than you can try [Laragon](https://laragon.org/docs/), It works very well for windows.

Closing it out.

Thank you.

**Comment by khanalpride** [comment]
@dvmunjapara yeah I was using Laragon long time ago, until I knew laravel horizon does not work on windows because of some ext not available on windows.
So I just switched to ubuntu desktop OS as a dual boot, and was all good there. But I was having to switch to windows time and again.

The I checked WSL and it was looking good. I tried it and everything was good.
I found this package https://github.com/valeryan/valet-wsl
which is fork of valet linux and works pretty well for me.

But looks like this valet-linux-plus package are both similar and using almost similar codebase. And it seems valet-linux-plus has same problem which valet-wsl fixed from official valet linux package.
